src/components/AddTask.js

```diff
@@ -70,7 +71,7 @@ const AddTask = () => {
     const addSubtask = (subtask) => {
         console.log(subtasks);
         if(!subtasks.includes(subtask)){
```

Comment:
Did you try adding a subtask with the same name?

This line probably doesn't work.